tests/foreman/ui/test_contentview.py

```diff
@@ -106,8 +106,13 @@ def setup_to_create_cv(self, repo_name=None, repo_url=None, repo_type=None,
                 reposet=rh_repo['reposet'],
                 releasever=rh_repo['releasever'],
             )
+        old_task_timeout = entity_mixins.TASK_TIMEOUT
+        # Update timeout to 10 minutes to finish sync
+        entity_mixins.TASK_TIMEOUT = 600
         # Sync repository
         entities.Repository(id=repo_id).sync()
+        entity_mixins.TASK_TIMEOUT = old_task_timeout
```

Comment:
I know, these lines were picked up directly from master with no changes. Do you want me to fix in 6.2.z only or in separate pr to master then cherry-pick to 6.2.z?